src/js/components/AppListComponent.jsx

```diff
@@ -28,20 +28,30 @@ function getGroupStatus(status, app) {
   return status;
 }
 
-function getGroupHealth(health, app) {
-  if (health == null) {
-    return app.health;
+function getGroupHealth(groupHealth, app) {
+  if (groupHealth == null) {
+    groupHealth = [];
   }
 
-  return health.map(healthState => {
-    var appHealthState =
-      app.health.find(appHealth => appHealth.state === healthState.state);
+  if (app.health == null) {
+    return groupHealth;
+  }
+
+  let appHealth = app.health;
 
-    if (appHealthState != null) {
-      healthState.quantity += appHealthState.quantity;
+  return appHealth.map(appHealthState => {
+    var groupHealthState =
```

Comment:
```groupHealthState``` needs a different name, otherwise eslint is complaining about same naming in different scopes.